DataFormats/L1TParticleFlow/BuildFile.xml

```diff
@@ -0,0 +1,8 @@
+<use   name="DataFormats/L1Trigger"/>
+<use   name="DataFormats/L1TrackTrigger"/>
+<use   name="DataFormats/L1THGCal"/>
```

Comment:
is `DataFormats/L1THGCal` actually used ?

Reply:
can be removed.